### ISSUE TITLE
How do i load this for some reason I been doing training but not really getting what i am looking out of it more information inside

### ISSUE THREAD

**Issue by TheMindExpansionNetwork** [filliptm/ComfyUI-FL-AceStep-Training#4]
Hello you all so been trying to wrap my head around this to see if I am missing something

I attempted a couple trainings but some reason not really hearing it not sure if I have it hooked up to use lora

So i attempt to trian on lets say 4 psytrance songs with the token p5ytr4nc3 and append it and preprocess it looks all good.

then i attempt to turn off to do training and it is good attempted the standard 500 epocs and even 2000 

then i load the standard ace-1.5 aio that comfy uses as template

Take the safetensor and .config

hook up from model to lora to think it would work and clip hooked up 

Not sure what I am missing or if its just something in my training prompting 

Thnak you for the ramble as i keep trying to figure this out

**Comment by dermichima-oss** [comment]
same, i dont get any audible effect even after 3000 epochs (i tried different settings at and datasets.)

<img width="494" height="671" alt="Image" src="https://github.com/user-attachments/assets/dcc363f0-feb6-4ed2-9ca9-117a0ca96da7" />

it has a visible change (the top one is with lora, the bottom without, same tags and seed) and yes i use my custom lora tag. the loss seems fine. it goes down to 0.4 or something at 3k epochs, 0.6 at like 500. 

my dataset(s) are: 1 with 16 clips, 28-40 seconds, i also tried one with 328 tracks, at 140sec. still nothing.

**Comment by BlissiraHQ** [comment]
I just came across these nodes.  The description is something beyond my wildest dreams. Not tried yet, but reading these error reports that this might not actually be working is on a level almost as horrific as the Epstien files. Gonna give it a whirl but I am now expecting my hopes and dreams to be curb stomped into the fiery depths,

**Comment by TheMindExpansionNetwork** [comment]
> I just finished my 3rd attempt and I have a very different experience. Tried to bump LR to 0.0005,trained for 1250 epochs. And at 1.7 strength lora produced the exact same copy of the track that was in the dataset. Like 1 to 1.  And it did it only on one seed(42) for some reason, after changing it it became still very similar but not an exact copy. 

Thank you for this going to change some settings and see I wonder if that's issue just need to change the learning rate 

I'll give you all some feedback 

Just want endless hippie jams lol

**Comment by ClaraSanders** [comment]
I have the same problem, but mine is worse. I have an RTX 5080 with 16GB of RAM and an i9 with 64 GB, and it is extremely slow when training to the point that it is useless, not to mention my GPU and VRAM are at 100% most of the time, which is not good for my 3K computer. It took me six hours to train just 50 epochs. Do I need to change some specific parameters, or am I just going insane?
